geometric_features/aggregation/__init__.py

```diff
@@ -1,7 +1,7 @@
 from geometric_features.aggregation.ocean import basins, subbasins, \
     antarctic, greenland, ice_shelves, ismip6, arctic as arctic_ocean, \
     transport, arctic_transport, moc
-from geometric_features.aggregation.seaice import arctic as arctic_seaice
+from geometric_features.aggregation.seaice import qgreenland, arctic as arctic_seaice
```

Comment:
For consistency, I suggest importing `qgreenland` with a name to make it clear its for sea ice:
```suggestion
from geometric_features.aggregation.seaice import arctic as arctic_seaice
from geometric_features.aggregation.seaice import qgreenland as qgreenland_seaice
```
(and then adjusting below accordingly)